### PR TITLE
Apply PR drupal#597

### PR DESCRIPTION
**Pull request by sluc23**
Apply drupal PR https://github.com/civicrm/civicrm-drupal/pull/597 to drupal-8 project

### CODE REVIEW

File: drush/civicrm.drush.inc [hunk 1]
```diff
@@ -579,21 +585,43 @@ function civicrm_drush_help($section) {
...
   \Drupal::service('civicrm')->initialize();
+  $status = drush_get_option('status', FALSE);
+  $params = [
+    'options' => [
+      'limit' => 0,
+    ],
+  ];
+  if ($status) {
+    if (!in_array($status, ['installed', 'uninstalled', 'disabled'])) {
+      return drush_set_error('CIVICRM_INVALID_STATUS', dt("Extension status {$status} is invalid."));
```
Comment (demeritcowboy) [inline drush/civicrm.drush.inc]:
```suggestion
      return drush_set_error('CIVICRM_INVALID_STATUS', dt("Extension status @status is invalid.", ['@status' => $status]));
```

File: drush/civicrm.drush.inc [hunk 2]
```diff
@@ -86,8 +86,14 @@ function civicrm_drush_command() {
   ];
   $items['civicrm-ext-list'] = [
     'description' => "List of CiviCRM extensions enabled.",
-    'examples' => [
+    'options' => [
+      'status' => 'Filter by extension status (installed, uninstalled, disabled).',
+      'out' => 'Output type: "pretty" (STDOUT) by default, "json" (STDOUT)',
+    ],
+     'examples' => [
```
Comment (demeritcowboy) [inline drush/civicrm.drush.inc]: The style-checker doesn't seem to run in this repo but this looks off by one char.

**Comment by demeritcowboy** [comment]
Except for the above minor points this looks good.

**Comment by sluc23** [comment]
thanks @demeritcowboy for the review, rebased with your suggestions!

**Comment by demeritcowboy** [comment]
Great!

**Pull request merged**